### PR TITLE
Add Efficientnet Edge presets

### PR DESCRIPTION
**Pull request by pkgoogle**
Adds these presets:

```
efficientnet_el_ra_imagenet
efficientnet_em_ra2_imagenet
efficientnet_es_ra_imagenet
```

[Tested Colab](https://colab.sandbox.google.com/gist/pkgoogle/b5c0ed4a9f87e5cf41159b2e40fee36c/densenet-example-keras-hub.ipynb)

### CODE REVIEW

File: keras_hub/src/models/efficientnet/efficientnet_backbone.py [hunk 1]
```diff
@@ -100,6 +100,8 @@ def __init__(
         stackwise_squeeze_and_excite_ratios,
         stackwise_strides,
         stackwise_block_types,
+        stackwise_force_input_filters=[0] * 7,
```
Comment (divyashreepathihalli) [inline keras_hub/src/models/efficientnet/efficientnet_backbone.py:103]: add the args in docstring

File: keras_hub/src/models/efficientnet/efficientnet_presets.py [hunk 2]
```diff
@@ -26,4 +26,43 @@
...
+            "description": (
+                "EfficientNet-EdgeTPU Large model trained on the ImageNet 1k "
+                "dataset with RandAugment recipe."
+            ),
+            "params": 10589712,
+            "official_name": "EfficientNet",
+            "path": "efficientnet",
+            "model_card": "https://arxiv.org/abs/1905.11946",
+        },
+        "kaggle_handle": "kaggle://keras/efficientnet/keras/efficientnet_b1_ft_imagenet",
```
Comment (divyashreepathihalli) [inline keras_hub/src/models/efficientnet/efficientnet_presets.py:40]: add version number of the model in the kaggle handle.

File: keras_hub/src/models/efficientnet/fusedmbconv.py [hunk 3]
```diff
@@ -70,8 +70,10 @@ def __init__(
         data_format="channels_last",
         se_ratio=0.0,
         batch_norm_momentum=0.9,
+        batch_norm_epsilon=1e-3,
```
Comment (divyashreepathihalli) [inline keras_hub/src/models/efficientnet/fusedmbconv.py:73]: add the added args in docstring

File: keras_hub/src/models/efficientnet/mbconv.py [hunk 4]
```diff
@@ -23,8 +23,10 @@ def __init__(
         data_format="channels_last",
         se_ratio=0.0,
         batch_norm_momentum=0.9,
+        batch_norm_epsilon=1e-3,
```
Comment (divyashreepathihalli) [inline keras_hub/src/models/efficientnet/mbconv.py:26]: add docstring for these args

File: keras_hub/src/models/efficientnet/efficientnet_backbone.py (hunk 1 above)
Comment (pkgoogle) [reply keras_hub/src/models/efficientnet/efficientnet_backbone.py:112]: done

File: keras_hub/src/models/efficientnet/efficientnet_presets.py (hunk 2 above)
Comment (pkgoogle) [reply keras_hub/src/models/efficientnet/efficientnet_presets.py]: done

File: keras_hub/src/models/efficientnet/fusedmbconv.py (hunk 3 above)
Comment (pkgoogle) [reply keras_hub/src/models/efficientnet/fusedmbconv.py:76]: done

File: keras_hub/src/models/efficientnet/mbconv.py (hunk 4 above)
Comment (pkgoogle) [reply keras_hub/src/models/efficientnet/mbconv.py:26]: done

**Comment by pkgoogle** [comment]
The remaining test appears irrelevant

**Review by divyashreepathihalli — APPROVED** [review]

**Pull request merged**